tensorflow_cloud/deploy.py

```diff
@@ -23,6 +23,14 @@
 from googleapiclient import errors
 
 from . import gcp
+from . import machine_config
+
+try:
+    from tensorflow.python.framework.versions import VERSION
+except ImportError:
+    # Use TF runtime version 2.1 (latest supported) as the default.
```

Comment:
2.2 not supported?

Reply:
Yes, 2.2 is not supported yet for cloud TPU training using AI platform. It is coming soon.